openshift/default-catalog-consistency/Makefile

```diff
@@ -0,0 +1,5 @@
+.PHONY: test-catalog
+test-catalog: ## Run the set of tests to validate the quality of catalogs
+	WHAT=catalog \
```

Comment:
Where is `WHAT` used?

Reply:
I believe that it will be used in the periodics/OCP CI ecosystem identification, so I can confirm its usage when I am doing the next step to integrate. If I see that is not required I can circle back and remove. 
I just want to keep to facilitate the integration.